### PR TITLE
Add edit id for commands

### PR DESCRIPTION
**Pull request by Vacxe**
Purpose:
For example we want to upload APK to `internal` testing channel. We should do it in the same transaction (*edit id*). Unfortunately we can't upload it to any channel without this chain. Also any transaction should be committed. 

```
edit create --config service_account.json --package-name com.any.package
apk upload --config service_account.json --package-name com.any.package --edit-id <my-transaction-id> --apk app-release.apk
tracks update --config service_account.json --package-name com.any.package --edit-id <my-transaction-id> --track "internal" --apk-version-code <my-version-code>
edit validate --config service_account.json --package-name com.any.package --edit-id <my-transaction-id>
edit commit --config service_account.json --package-name com.any.package --edit-id <my-transaction-id>
```

### CODE REVIEW

File: src/main/kotlin/com/github/vacxe/googleplaycli/Commands.kt [hunk 1]
```diff
@@ -38,206 +47,240 @@ object Commands {
...
+            val language: String by option("--language", "-l", help = "The language code (a BCP-47 language tag) of the localized listing to read or modify. For example, to select Latin American Spanish, pass \"es-419\".").required()
+
+            override fun run(api: PlayStoreApi) = api.listingsDeleteAll(EditModel(packageName, editId))
+        }
+
+        class List : EditCommand(name = "list", actionDescription = "Returns all of the localized store listings attached to this edit") {
             val language: String by option("--language", "-l", help = "The language code (a BCP-47 language tag) of the localized listing to read or modify. For example, to select Latin American Spanish, pass \"es-419\".").required()
 
-            override fun run(api: PlayStoreApi) = api.listingsGet(ListingsGetModel(packageName, language))
+            override fun run(api: PlayStoreApi) = api.listingsList(EditModel(packageName, editId))
```
Comment (aafanasev) [inline src/main/kotlin/com/github/vacxe/googleplaycli/Commands.kt:174]: `language` variable is not used, is it intentional?

File: src/main/kotlin/com/github/vacxe/googleplaycli/Commands.kt [hunk 2]
```diff
@@ -38,206 +47,240 @@ object Commands {
...
+            override fun run(api: PlayStoreApi) = api.imagesUploadAll(ImagesUploadModel(packageName, editId, imageType, language, image))
         }
     }
 
     object Listings {
-        class Get : BaseCommand(name = "get", actionDescription = "Fetches information about a localized store listing") {
+        class DeleteAll : EditCommand(name = "deleteAll", actionDescription = "Deletes all localized listings from an edit") {
+            val language: String by option("--language", "-l", help = "The language code (a BCP-47 language tag) of the localized listing to read or modify. For example, to select Latin American Spanish, pass \"es-419\".").required()
+
+            override fun run(api: PlayStoreApi) = api.listingsDeleteAll(EditModel(packageName, editId))
```
Comment (aafanasev) [inline src/main/kotlin/com/github/vacxe/googleplaycli/Commands.kt:168]: `language` variable is not used, is it intentional?

File: src/main/kotlin/com/github/vacxe/googleplaycli/actions/Tracks.kt [hunk 3]
```diff
@@ -10,40 +10,40 @@ import com.google.api.services.androidpublisher.model.TrackRelease
...
-                })
-        )
-        return edits.tracks().update(model.packageName, insert.id, model.track, track).execute()
+        val release = TrackRelease()
+                .setVersionCodes(listOf(model.apkVersionCode.toLong()))
+                .setStatus("completed")
+
+        val track = Track().setReleases(listOf(release))
+
+        return edits.tracks().update(model.packageName, model.editId, model.track, track).execute()
```
Comment (aafanasev) [inline src/main/kotlin/com/github/vacxe/googleplaycli/actions/Tracks.kt:47]: shouldn't this have the same logic as others `model.editId ?: edits.insert(model.packageName, null).execute()`? as I see `TracksUpdateModel#editId` is nullable

**Comment by Vacxe** [comment]
@aafanasev thanks for the review

**Pull request merged**